packages/metal-quartz-icon/src/MetalQuartzIcon.js

```diff
@@ -20,25 +19,33 @@ class MetalQuartzIcon extends Component {
  */
 MetalQuartzIcon.STATE = {
 	/**
-	 * Path to the spritemap file with the icons.
+	 * CSS classes to be applied to the element, use this to pass in classes for
+	 * third-party icon libraries e.g. `glyphicon glyphicon-plus`. If
+	 * elementClasses is defined and spritemap or symbol is undefined, it will
+	 * insert elementClasses into a span tag.
+	 * @type {string}
+	 */
+	elementClasses: {
```

Comment:
`elementClasses` is actually a default property on `Component`, so we don't need to add it to anything extending from it.